test/controllers/private_messages_controller_test.rb

```diff
@@ -0,0 +1,16 @@
+require 'test_helper'
+
+class PrivateMessagesControllerTest < ActionController::TestCase
+  setup do
+
+    @ticket = tickets(:problem)
+    @reply = replies(:solution)
+
+    sign_in users(:alice)
+  end
+
+  test 'create private message' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.